html/tests/tabs.tests.js

```diff
@@ -105,74 +101,76 @@ describe('tabs UI Events tests', () => {
   });
 
   it('should bind the click event on each tab', () => {
-    expect(tab1.addEventListener.getCall(0).args[0]).eql('click');
-    expect(tab2.addEventListener.getCall(0).args[0]).eql('click');
-    expect(tab3.addEventListener.getCall(0).args[0]).eql('click');
+    expect(tab1.addEventListener.getCall(0).args[0]).toBe('click');
+    expect(tab2.addEventListener.getCall(0).args[0]).toBe('click');
+    expect(tab3.addEventListener.getCall(0).args[0]).toBe('click');
   });
 
   it('should bind the keydown event to the container', () => {
-    expect(tabContainer.addEventListener.getCall(0).args[0]).eql('keydown');
+    expect(tabContainer.addEventListener.getCall(0).args[0]).toBe('keydown');
   });
 
   it('should activate the previous tab when left is pressed', () => {
     event = new window.Event('keydown');
     event.keyCode = 37;
     tab2.click();
-    expect(tab2.classList.contains('sprk-c-Tabs__button--active')).eql(true);
+    expect(tab2.classList.contains('sprk-c-Tabs__button--active')).toBe(true);
     tabContainer.dispatchEvent(event);
-    expect(tab1.classList.contains('sprk-c-Tabs__button--active')).eql(true);
+    expect(tab1.classList.contains('sprk-c-Tabs__button--active')).toBe(true);
   });
 
   it('should activate the next tab when right is pressed', () => {
     event = new window.Event('keydown');
```

Comment:
`event = new Event('keydown');` ?

Reply:
window.Event works since I added it to jest globals, so this way or the other way works.